.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - 'stable'
+install:
+  - npm install
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database authors_haven_test;' -U postgres
+
```

Comment:
add a command to build the app.

Reply:
I just put it here
<img width="191" alt="Screen Shot 2019-07-31 at 14 11 52" src="https://user-images.githubusercontent.com/39116572/62210791-44fa4300-b39d-11e9-84bb-8dfff99fedf3.png">